### PR TITLE
Add methods for printing proper characters

### PR DESCRIPTION
**Pull request by radvk**

### CODE REVIEW

File: printer.rb [hunk 1]
```diff
@@ -1,12 +1,13 @@
+require_relative 'player'
 require 'curses'
 include Curses
+require 'pry'
```
Comment (bartoszmaka) [inline printer.rb]: ✂️

File: printer.rb [hunk 2]
```diff
@@ -28,4 +29,22 @@ def call(board)
...
+
+  def print_proper_sign(element)
+    if NUMBER_TO_CHAR_MAP.include?(element)
+      addstr(NUMBER_TO_CHAR_MAP[element])
+    else
+      addstr(element.display)
+    end
+  end
+
+  def print_proper_sign_2(el)
```
Comment (dominikduda) [inline printer.rb]: Where is it used?

Comment (radvk) [reply printer.rb]: We wrote both methods for practice, you can throw away one of them, you can choose which one.

Comment (dominikduda) [reply printer.rb]: Remove this one then, please.

**Comment by dominikduda** [comment]
Code looks ok, but it **does not work**.

image

Pressing 'w' or 's' does nothing (player movement is already implemented). Please investigate. If fixing it requires changes outside of `Printer` then ping me.

**Pull request merged**